src/main/java/gregtech/loaders/recipe/chemistry/ChemistryRecipes.java

```diff
@@ -103,5 +103,23 @@ public static void init() {
                 .fluidOutputs(EnderAir.getFluid(10000))
                 .dimension(1)
                 .duration(200).EUt(256).buildAndRegister();
+
+        // CaCO3 + 2NaCl -> Na2CO3 + CaCl2
+        BLAST_RECIPES.recipeBuilder()
+                .input(dust, Calcite, 5)
+                .input(dust, Salt, 4)
+                .output(dust, SodaAsh, 6)
+                .output(dust, CalciumChloride, 3)
+                .duration(120).EUt(VA[MV]).blastFurnaceTemp(500)
```

Comment:
This temperature could probably be bumped to somewhere between `1200` and `1700`.